source/distributed/tria.cc

```diff
@@ -3880,13 +4211,33 @@ namespace parallel
                static_cast<unsigned int>(parallel_forest->local_num_quadrants),
              ExcInternalError());
 
-      // deregister pack_callback function
+#  ifdef DEBUG
+      // check validity of handle and deregister pack_callback function.
       // first reset with invalid entries to preserve ambiguity of
-      // handles, then free memory when all were unpacked (see below)
-      // consider_cell_invalid is irrelevant in this context
-      Assert(cell_attached_data.pack_callbacks[handle] != nullptr,
-             ExcInternalError());
-      cell_attached_data.pack_callbacks[handle] = nullptr;
+      // handles, then free memory when all were unpacked (see below).
+      const unsigned int callback_index = std::floor(handle / 2);
```

Comment:
you don't need the `std::floor` call here: `handle` and `2` are both integral types, so the division happens in integer arithmetic